src/CarbonAware.CLI/src/Commands/Emissions/EmissionsCommand.cs

```diff
@@ -0,0 +1,57 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.CLI.Common;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.Commands.Emissions;
+
+class EmissionsCommand : Command
```

Comment:
Suggestion: Maybe to have a base class that has Location. Seems like all the commands require it.

Reply:
Part of the design intention here is that each command class is fully self-describing.  IE you can read the single class file and understand the args, options, subcommands that it supports.

I don't know where exactly to communicate that type of intention.  Any thoughts?  Maybe `/docs/architecture/cli.md` ?


But again, I think that our implementation philosophy here is to wait until there is a second class that duplicates it and make the decision at that time when we can see the bigger picture.

Reply:
@bderusha , I agree that for this 1st iteration, better to keep it simple. I'm not 100% sure that adding `cli.md` would help, until another command is implemented to know better what would be. IMHO, I would suggest to have it as a draft somewhere else.